project/classic/aem/script/undeploy.sh

```diff
@@ -0,0 +1,3 @@
+step "undeploy AEM dispatcher"
```

Comment:
This name is kinda funny and misleading too. Do we even need it? I'd move it to "down.sh" and get rid of undeploy

Reply:
I wanted to have things paired conceptually..

down is a more general hook, but yes, I have same doubt here; I will rethink it :)

Reply:
down was designed to control only aem; undeploy probably for the rest things; maybe naming could be polished somehow but I have no idea for now